ukaea/fair-mast-ingestion
# Add mappings to rename groups

**Cannot be merged before** #39 

This adds mappings to rename groups from short names to long names and add an IMAS attribute.

To test:
 - Run ingestion and check the names of the groups and attributes
 - Code review

Closes #43
